### ISSUE TITLE
Add search option to all tables

### ISSUE THREAD

**Issue by wszarmach** [krico/jas#442]
For example,
* Activities
* Activity Packages
* Activity Types
* Organizations (with a z hmpf)
* Groups
* Balance

**Comment by pauljanes** [comment]
Yeah organiZations is the American English spelling and organiSations the British English one. I dont really mind which but then we should be consistent throughout jasify. To be honest American English seems to be becoming more and more popular than British English.

**Comment by wszarmach** [comment]
I can not believe you said that! You better hope the queen doesn't see your comments :crown:
